Tentacle/Milestone.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  Milestone.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-05-23.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Foundation
+import Argo
+import Curry
+
+public struct Milestone: Hashable, CustomStringConvertible {
+    public enum State: String {
+        case Open = "open"
+        case Closed = "closed"
+    }
+
+    /// The ID of the milestone
+    public let ID: String
+
+    /// The number of the milestone in the repository it belongs to
+    public let number: Int
+
+    /// The state of the Milestone, open or closed
+    public let state: State
+
+    /// The title of the milestone
+    public let title: String
+
+    /// The description of the milestone
+    public let contentDescription: String
+
+    /// The user who created the milestone
+    public let creator: User
+
+    /// The number of the open issues in the milestone
+    public let numberOfOpenIssues: Int
+
+    /// The number of closed issues in the milestone
+    public let numberOfClosedIssues: Int
```

Comment:
I took another look at the Swift stdlib and I think we should make these `*Count`.

Maybe `openIssueCount` and `closedIssueCount`?

Sorry. I know you've already changed this once.